src/gc/proxy.d

```diff
@@ -13,13 +13,16 @@
  */
 module gc.proxy;
 
-import gc.gc;
+import conservative = gc.impl.conservative.gc;
+import malloc = gc.impl.malloc.gc;
```

Comment:
We shouldn't use that name in the real of memory management, rather call it manual or so.